modal/_output.py

```diff
@@ -129,6 +129,7 @@ class OutputManager:
     _line_buffers: dict[int, LineBufferedOutput]
     _status_spinner: Spinner
     _app_page_url: Optional[str]
+    is_app_running: bool
```

Comment:
i'm not a big fan of this since
1. it couples the output mgr's behavior to the app – why does the output mgr need to know about the app's state? it would make more sense if it's just a flag `show_image_logs` or similar
2. mutating state, but i can't think of any way to avoid it so 🤷

Reply:
We could do a flag but then we'd have to launch the logs loop twice (once while creating the objects, and the other while running). I believe we used to do something like that?

Reply:
can we just rename it to `_show_image_logs` or similar? otherwise LGTM!!

Reply:
done